packages/orm/src/test/functional/many2many.ts

```diff
@@ -8,15 +9,20 @@ import {
   TagProps,
 } from "../helpers";
 
-class User extends Model {
+const registry = ModelDescriptorsRegistry.getInstance();
```

Comment:
In general, the structure of unit tests should be changed. We are using now a global storage, cleaning up the registry and creating classes should be wrapped into a function block.

Reply:
Totally agree.